src/app/courses/components/edit-course-page/edit-course-page.component.html

```diff
@@ -1,15 +1,13 @@
-<p>Title:</p>
-  <input type="text" id="name" [(ngModel)]="name" required>
-  <p>Description:</p>
-  <textarea rows="6" ngModel [(ngModel)]="description" required></textarea>
-    <p>Date:</p>
-  <input type="date" [disabled]="id" [(ngModel)]="date" required>
-<div class="duration">
-  <p>Duration:</p>
-  <input type="number" [(ngModel)]="length" required>
-  <p *ngIf="length">{{length | durationPipe}}</p>
-</div>
+<form [formGroup]="groupControl" (ngSubmit)="onSave()">
+<p>Title:</p><p class="errorText" *ngIf="validationTitle()">Max length: {{maxTitle}}</p>
+  <input formControlName="nameControl" [class.error]="validationTitle()" type="text" id="name" required >
```

Comment:
Even though the field is required, there is no error message shown when it's empty. Do  you wanna add one? Because now it's not obvious why "Save" button is disabled when title or description fields are empty.

Reply:
good point, done